HTML/index.html

```diff
@@ -6,25 +6,32 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="description" content="Delicious' Chocolate Pizza recipe">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href='http://fonts.googleapis.com/css?family=Yellowtail' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Oswald:b' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Abel' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Patrick+Hand' rel='stylesheet' type='text/css'>
+  <link href='http://fonts.googleapis.com/css?family=Droid+Serif' rel='stylesheet' type='text/css'>
+
+
   <link href="style.css" type="text/css" rel="stylesheet"/>
```

Comment:
Remember in the href you'll need to travel to the directory your css is in. So, if the index.html is in an HTML file (it would be better to put it in the root directory, do:
```
<link href="../CSS/style.css" type="text/css" rel="stylesheet">
```
from root directory:
```

<link href="css/style.css" type="text/css" rel="stylesheet">